src/protocols/dhcpv4/encode.c

```diff
@@ -151,13 +142,13 @@ static uint8_t *extend_option(uint8_t *start, uint8_t *end, uint8_t *p, int len)
 		 *	option, and how much data we have to move out
 		 *	of the way.
 		 */
-		sublen = 255 - out[1];
+		sublen = 255 - hdr_field[1];
 		if (sublen > len) sublen = len;
 
 		/*
 		 *	Add in the data left at the current pointer.
 		 */
-		out[1] += sublen;
+		hdr_field[1] += sublen;
```

Comment:
we should just use `out` here.  there's no need to rename the variables to `hdr_field`.  it doesn't match the style of the rest of the code.